launch/tier4_control_launch/launch/control.launch.py

```diff
@@ -306,6 +306,13 @@ def launch_setup(context, *args, **kwargs):
         target_container="/control/control_container",
     )
 
+    glog_component = ComposableNode(
+        package="glog_component",
+        plugin="glog_component::GlogComponent",
```

Comment:
@TakaHoribe The plugin seems to be registered as `GlogComponent`, so I guess you should add namespace into component implementation or remove namespace here. Please check if this component is loaded by this launch.

Reply:
Thank you! I forgot to push some local commits. Not it should work!